rotkehlchen/chain/ethereum/manager.py

```diff
@@ -620,7 +622,7 @@ def _ens_lookup(
 
         address = self._call_contract(
             web3=web3,
-            contract_address=to_checksum_address(resolver_addr),
+            contract_address=deserialize_ethereum_address(resolver_addr),
```

Comment:
Here and in line 636 you need to handle the potential `DeserializationError`. Which I don't think can actually ever happen unless somehow the contract returns invalid address ... but well being defensive can happen if somehow contract changes underneath. In that case I would log an error and return `None`.